networkx/generators/classic.py

```diff
@@ -207,6 +208,22 @@ def circular_ladder_graph(n,create_using=None):
     G.add_edge(n,2*n-1)
     return G
 
+
+def circulant_graph(n,offsets,create_using=None):
+    """Return the circulant graph Ci_n(x_1, x_2, ..., x_m) with n vertices.
+
+    Ci_n(x_1, ..., x_m) consists of n vertices 0, ..., n-1 such that the vertex
+    with label i is connected to the vertices labelled (i + x) and (i - x),
+    for all x in x_1 up to x_m, taken modulo n.
+    """
+    G=empty_graph(n,create_using)
```

Comment:
Needs spaces around assignment operator, and after comma. Same with the line below.